helm_deploy/offender-case-notes/values.yaml

```diff
@@ -43,7 +43,7 @@ generic-service:
       DATABASE_PASSWORD: "database_password"
       DATABASE_NAME: "database_name"
       DATABASE_ENDPOINT: "rds_instance_endpoint"
-    domain-events-topic:
+    hmpps-domain-events-topic:
```

Comment:
both of these appear in the deployment (via CP) I will do the CP PRs once this has deployed